core/predicates/predicates.go

```diff
@@ -12,98 +12,103 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Package predicates is a utility package to create predicates.
 package predicates
 
 import (
 	. "github.com/hazelcast/hazelcast-go-client/internal/predicates"
 )
 
-// Sql is a helper function for creating SqlPredicate.
+// Sql returns a SqlPredicateSql with the given sql.
```

Comment:
These changes are also unnecessary. There is nothing wrong at previous one.

Reply:
We have used `returns` pattern for all `New` functions so we should keep it consistent.